contribs/gmf/src/directives/layertree.js

```diff
@@ -181,14 +179,42 @@ gmf.LayertreeController = function($http, $sce, $scope, ngeoCreatePopup,
 
 
 /**
+ * LayertreeController.prototype.prepareLayer_ - inject metadata into the layer
+ * @private
+ * @param {GmfThemesNode} node Layer tree node.
+ * @param {ol.layer.Base} layer The OpenLayers layer or group for the node.
+ */
+gmf.LayertreeController.prototype.prepareLayer_ = function(node, layer) {
+  var type = gmf.Themes.getNodeType(node);
+  var ids = this.getNodeIds_(node);
+  if (goog.isDefAndNotNull(layer)) {
```

Comment:
if the layer can be `null` or `undefined`, it should be defined as such in the param section above, as such:

`@param {?ol.layer.Base=} opt_layer The...`